data/xml/1952.preconf.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
The OG site mentions `summarized in Reifler’s report` and ` summarised in Reynold’s report]` for papers without a pdf. I wonder if we have these reports? 

Otherwise the 1952 xml LGTM

Reply:
I don't think we have those unfortunately :(